htdocs/theme/eldy/global.inc.php

```diff
@@ -3096,23 +3079,27 @@
 /* ============================================================================== */
 
 div.vmenu, td.vmenu {
-	margin-<?php print $right; ?>: 2px;
 	position: relative;
 	float: left;
-	padding: 0px;
-	padding-bottom: 0px;
-	padding-top: 1px;
-	width: 190px;
+	width: 200px;
```

Comment:
If you made some changes into CSS, does it change visual effect ? If yes, can you provide a screenshot of after (on desktop and smartphone) ?

Reply:
Okay I see, thanks!
I am working on a taskboard to reference IHM integration method,
let me try something and I’ll come back to you :)